Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/Color.h

```diff
@@ -29,3 +30,5 @@ namespace ScriptCanvas
         ColorType One();
     } // namespace ColorFunctions
 } // namespace ScriptCanvas
+
+#include <Include/ScriptCanvas/Libraries/Math/Color.generated.h>
```

Comment:
missing newline at end of the file
```suggestion
#include <Include/ScriptCanvas/Libraries/Math/Color.generated.h>

```